app/components/header.module.css

```diff
@@ -58,6 +58,12 @@ input.search {
     background-size: 14px 15px;
     border-radius: 15px;
     box-shadow: none;
+    transition: box-shadow 150ms;
+
+    &:focus {
+        outline: none;
```

Comment:
does this not mess with accessibility?

Reply:
only if there is no visual alternative, which the shadow provides in this case